src/app/mail/mail-sidebar/compose-mail-dialog/compose-mail-dialog.component.ts

```diff
@@ -49,6 +49,22 @@ export class ComposeMailDialogComponent {
     this.hideMailComposeDialog();
   }
 
+  toggleMinimized() {
```

Comment:
I don't understand your logic on minimize and maximize, why to use two variables?
It is just a simple boolean,
it is either minimized or maximized.